lib/mutant/matcher/config.rb

```diff
@@ -22,6 +38,45 @@ def add(attribute, value)
         update(attribute => public_send(attribute).dup << value)
       end
 
+    private
+
+      # Present attributes
+      #
+      # @return [Array<Symbol>]
+      #
+      # @api private
+      def present_attributes
+        to_h.reject { |_key, value| value.empty? }.map(&:first)
+      end
+
+      # Formatted attributes
+      #
+      # @return [String]
+      #
+      # @api private
+      def inspect_attributes
+        attributes = present_attributes
+          .map(&method(:format_attribute))
+          .join(ATTRIBUTE_DELIM)
+
+        return EMPTY_ATTRIBUTES if attributes.empty?
```

Comment:
I think this guard clause doesn't really protect very complex code, I would probably write this as:

```ruby
attributes.empty? ? EMPTY_ATTRIBUTES : attributes
```